### PR TITLE
use model validator instead

### PR DESCRIPTION
**Pull request by mochic**
Addresses #804

### CODE REVIEW

File: src/aind_data_schema/core/rig.py [hunk 1]
```diff
@@ -78,41 +78,40 @@ class Rig(AindCoreModel):
         description="Path to file that details the CCF-to-lab coordinate transform",
     )
     origin: Optional[Origin] = Field(None, title="Origin point for rig position transforms")
-    rig_axes: Optional[List[Axis]] = Field(default=[], title="Rig axes", min_length=3, max_length=3)
+    rig_axes: Optional[List[Axis]] = Field(None, title="Rig axes", min_length=3, max_length=3)
```
Comment (dyf) [inline src/aind_data_schema/core/rig.py:81]: any reason your are changing the default here?

**Review by dyf — CHANGES_REQUESTED** [review]

File: src/aind_data_schema/core/rig.py (hunk 1 above)
Comment (mochic) [reply src/aind_data_schema/core/rig.py:81]: Yeah. Sorry this is included in this pr and also not in it's own commit. I think that the current default is incorrect and causes future deserialization issues. It is a list that doesn't match the min length requirement for the field. I can remove it as it is out of the scope of this pr.

Comment (mochic) [reply src/aind_data_schema/core/rig.py]: Resolved in 1ce710e but I think that this change needs to occur at some point to allow for models to be serialized with the default and then be deserialized with later issues.

**Comment by mochic** [comment]
@dyf I added a new test to supplement the loss of coverage but it does bother me that this change is significant enough that what was covering this in the past no longer is.

**Comment by dyf** [comment]
@mochic I was just looking at this issue - is there a reason you closed it?

**Comment by mochic** [comment]
@dyf I think that the bug I was experiencing was related to something else on my end. It's possible that model validators are more "correct" but I don't think we get anything out of it.

**Pull request closed without merging**